controllers/errorHandler.js

```diff
@@ -0,0 +1,14 @@
+
+// eslint-disable-next-line no-unused-vars
+const errorHandler = (error, req, res, next) => {
+  const status = error.status || 500;
+  const message = status === 500 ? 'Internal Server Error' : error.message;
+  res.status(status).send({
+    error: {
+      status,
+      message,
+    },
+  });
```

Comment:
Можно лучше: не дублировать код ошибки в ответе.
Код ответа будет доступен в отдельном заголовке в ответе. Описание ошибки нужно, чтобы добавить больше детальной информации о том, почему произошла ошибка. Достаточно оставить так:
```suggestion
  res.status(status).send({ message });
```